### DISCUSSION TITLE
Adding directories to picker and resource

### DISCUSSION THREAD

**Discussion by vovacorgo** [awcodes/filament-curator#284] [Ideas]
Can you add separating and adding directories to curator (to picker and for the resource), Because if you have much of photos you cant find properly image. That will be good for more massive project

**Comment by vovacorgo** [comment]
You can firstly add only seeing of directories and open them, later creating

**Reply by awcodes** [reply]
What?

**Reply by Wulfheart** [reply]
@awcodes Even though he phrased it in a way that is not Easy to understand in my opinion, this would be a great addition. I would love to be able to group the images/files in some directories so I can find them more easily later.
